core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala

```diff
@@ -61,17 +63,17 @@ trait Implicits {
     val self: RDD[(SpaceTimeKey, T)] with Metadata[TileLayerMetadata[SpaceTimeKey]]
   )(implicit spark: SparkSession) extends SpatioTemporalContextRDDMethods[T]
 
-  private[astraea]
+  private[rasterframes]
```

Comment:
Interesting but does not change the scope, correct?

Reply:
It was `astraea` before so I could access them from our other code.... but that was a sneaky hack. So this does technically change the scope, but only in a way that forces me to "do the right thing" elsewhere.

Reply:
👍